Common/Charting.cs

```diff
@@ -178,18 +177,18 @@ public Series GetUpdates()
                 //Shuffle the update point to now:
                 _updatePosition = Values.Count;
             }
-            catch (Exception err) {
+            catch (Exception err)
+            {
                 Log.Error("Series.GetUpdates(): " + err.Message);
             }
             return copy;
         }
     }
 
-
     /// <summary>
     /// Single Chart Point Value Type for QCAlgorithm.Plot();
     /// </summary>
-    [JsonObjectAttribute]
+    [JsonObject]
```

Comment:
Any reason for this change? I'm not 100% it will behave the same. It could break the charting system if it changes behaviour